ABLTemporalBoneSegmentation.json

```
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-item-schema-v1.0.0.json#",
```

@jcfr Is this link needing to be changed?
```suggestion
  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-entry-schema-v1.0.0.json#",
```

---

Good catch

---

Same issue at https://github.com/Slicer/Slicer/blob/1b9bd54b06fa1b237a0caa8399c0a50de5b19283/Schemas/slicer-extension-catalog-entry-schema-v1.0.0.json#L3

---

See https://github.com/Slicer/Slicer/pull/7710